package.json

```diff
@@ -27,6 +27,16 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "react-native": ">=0.47"
-  }
+    "react-native": "<0.62",
+    "react-native-windows": "^0.61.9"
```

Comment:
Any reason why `"react-native": "<0.62"`?

This file should stay as it was since to develop this new feature, no changes are required to any of the dependencies (`peerDependencies`, `devDependencies`, `dependencies`).